.buildkite/pipelines/serverless_deployment/create_deployment_tag.yml

```diff
@@ -33,5 +33,9 @@ steps:
       - ts-node .buildkite/scripts/serverless/create_deploy_tag/release_wizard_messaging.ts --state create_deploy_tag
       - bash .buildkite/scripts/serverless/create_deploy_tag/create_deploy_tag.sh
       - ts-node .buildkite/scripts/serverless/create_deploy_tag/release_wizard_messaging.ts --state tag_created
+      - ts-node .buildkite/scripts/serverless/create_deploy_tag/generate_gpctl_trigger.ts
```

Comment:
yes, normally. 
However, the trigger parameters are not available at the time when the pipeline is evaluated. This is why we have to emit, because only through the selection process would we have the parameters ready for the trigger.

Reply:
Does that mean we can't simply follow the flow using the Buildkite UI by clicking on the steps to go to the next pipeline?

Reply:
We will be able to, the newly added step will appear as a step in the buildkite task - the only thing is we need to dynamically upload the step if we don't have all the variables at hand at the initial upload.